packages/snippet-bot/package-lock.json

```diff
@@ -0,0 +1,6498 @@
+{
```

Comment:
I've just updated `package.json` in this directory. Do you need to update the lock file with the `package.json` in master?

Reply:
Nope!  The robots will take care for us.